app/components/comment-item.js

```diff
@@ -1,5 +1,49 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-  classNames: ['comment-item']
+  classNames: ['comment-item'],
+  classNameBindings: ['isEditing:editing'],
+
+  session: Ember.inject.service(),
+
+  currentUserId: Ember.computed.alias('session.session.authenticated.user_id'),
+  commentAuthorId: Ember.computed.alias('comment.user.id'),
+  currentUserIsCommentAuthor: Ember.computed('currentUserId', 'commentAuthorId', function() {
+    let userId = parseInt(this.get('currentUserId'), 10);
+    let authorId = parseInt(this.get('commentAuthorId'), 10);
+    return userId === authorId;
+  }),
+
+  canEdit: Ember.computed.and('session.isAuthenticated', 'currentUserIsCommentAuthor'),
```

Comment:
Common to keep the aliases all together near the top after static attributes, then computed properties, then methods, then actions. Keeps the files clean and clear.